tailwind.config.js

```diff
@@ -1,7 +1,18 @@
+const plugin = require('tailwindcss/plugin');
+
 module.exports = {
   content: ['./output/*.html'],
   theme: {
     extend: {},
   },
-  plugins: [],
+  plugins: [
+    plugin(({ addUtilities }) => {
```

Comment:
It might be nice to add some commentary here to make it clear that these are utility classes being added by Ably. Or, even, if the name `addUtilities` is under our full control (i.e. does not need to be named exactly that) then perhaps rename to `addAblyUtilities`.

Reply:
Very good point. Updated this line to suggest this.